modules/packit-api.nix

```
@@ -192,10 +211,15 @@ in
         PACKIT_DEFAULT_ROLES = lib.concatStringsSep "," instanceCfg.defaultRoles;
         PACKIT_CORS_ALLOWED_ORIGINS = lib.concatStringsSep "," instanceCfg.corsAllowedOrigins;
         PACKIT_AUTH_METHOD = instanceCfg.authentication.method;
+        PACKIT_ORDERLY_RUNNER_ENABLED = if (instanceCfg.runner.enable && instanceCfg.runner.repositoryUrl != "") then "true" else "false";
```

```suggestion
        PACKIT_ORDERLY_RUNNER_ENABLED = if instanceCfg.runner.enable then "true" else "false";
```

IMO setting `enable = true` but not giving it a `repositoryUrl` is a bug in the caller of the module. I think it would be more appropriate if `multi-packit` did the check instead.

---

this is fair